modal_proto/api.proto

```diff
@@ -1471,6 +1471,7 @@ message FunctionCallCallGraphInfo {
 message FunctionCallCancelRequest {
   string function_call_id = 1;
   bool terminate_containers = 2;
```

Comment:
oops can we also add a comment that sync cancellations on the input plane do not support this field?